src/main/java/duke/DateParser.java

```diff
@@ -0,0 +1,24 @@
+package duke;
```

Comment:
Perhaps you can package this class into a parser package in Duke. 
```suggestion
package duke.parser;
```